From 377ad06ae08fb5135253122e68f0f1004f698a44 Mon Sep 17 00:00:00 2001
Message-Id: <377ad06ae08fb5135253122e68f0f1004f698a44.1424979244.git.feitong.yi@intel.com>
In-Reply-To: <d8466aca83b11ab8b0870e775536306782d73e9f.1424979244.git.feitong.yi@intel.com>
References: <d8466aca83b11ab8b0870e775536306782d73e9f.1424979244.git.feitong.yi@intel.com>
From: Shashank Sharma <shashank.sharma@intel.com>
Date: Thu, 5 Feb 2015 16:46:53 +0530
Subject: [PATCH 22/28] REVERTME [VPG]: video/adf: Disable PFI credits

This patch disables the PFI credits programming.

REVERTME: Programming the PFI credits during the
dual display hotplug is causing a hardware hang,
because of which this is disbaled. PFI credits programming
is to be done only when display pipe is disabled,
once this is handled this revert can be removed.

Issue: GMINL-5482
Change-Id: I98972ec922643d51d213f344a144bb552badd6aa
Signed-off-by: Shashank Sharma <shashank.sharma@intel.com>
---
 drivers/video/adf/intel/core/vlv/vlv_pipeline.c |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/video/adf/intel/core/vlv/vlv_pipeline.c b/drivers/video/adf/intel/core/vlv/vlv_pipeline.c
index 1552602..87a37a8 100644
--- a/drivers/video/adf/intel/core/vlv/vlv_pipeline.c
+++ b/drivers/video/adf/intel/core/vlv/vlv_pipeline.c
@@ -263,7 +263,9 @@ u32 vlv_pipeline_on(struct intel_pipeline *pipeline,
 		struct drm_mode_modeinfo *mode)
 {
 	struct vlv_pipeline *disp = to_vlv_pipeline(pipeline);
+#if 0
 	struct intel_dc_config *intel_config = &disp->config->base;
+#endif
 	struct vlv_pri_plane *pplane = &disp->pplane;
 	struct vlv_pipe *pipe = &disp->pipe;
 	struct vlv_plane_params plane_params;
@@ -372,8 +374,10 @@ u32 vlv_pipeline_on(struct intel_pipeline *pipeline,
 	if (err)
 		pr_err("ADF: %s: program pipe failed\n", __func__);
 
+#if 0
 	/* program PFI credits before pipe is enabled */
 	vlv_pm_update_pfi_credits(intel_config);
+#endif
 
 	/* pipe enable */
 	err = vlv_pipe_enable(pipe, mode);
-- 
1.7.9.5

